protocols/v2/const-sv2/README.md

```diff
@@ -0,0 +1,21 @@
+# const_sv2
+
+[![crates.io](https://img.shields.io/crates/v/const_sv2.svg)](https://crates.io/crates/const_sv2)
+[![docs.rs](https://docs.rs/const_sv2/badge.svg)](https://docs.rs/const_sv2)
+
+`const_sv2` is a Rust crate that provides essential constants for the SV2 (Stratum V2) protocol. These constants are crucial for message framing, encryption, and protocol-specific identifiers across various SV2 components, including Mining, Job Declaration, and Template Distribution protocols.
+
+## Features
```

Comment:
I usually feel the word "Feature" can be confusing, because in the context of Rust it usually means "feature flag" for `cargo`.

Reply:
Done here d84c53cc91b97fae08c5b28f90ecd16a7cea9254